Assets/PlayroomKit/PlayroomKit.cs

```diff
@@ -971,6 +970,41 @@ public class Dpad
             public string y;
         }
 
+
+        // RPC:
+        public enum RPCModes
+        {
+            ALL,
+            OTHERS,
+            HOST
+        }
+
+        [DllImport("__Internal")]
+        public extern static void RPCregister(string name);
```

Comment:
Minor nit: function naming convention

Reference: https://chat.openai.com/share/ea8c8c00-eb88-48ed-ba62-0967a7b48b50

Reply:
My bad, will update to PascalCase for consistency.